Engine/libsrc/apeg-1.2.1/audio/mpg123.c

```diff
@@ -10,7 +10,7 @@
 #endif
 
 #ifndef __GNUC__
-#define snprintf _snprintf
+#include "util/c99_snprintf.h"
```

Comment:
isn't this one of the files scheduled for deletion in @ivan-mogilko 's cleanup PR ?